docs/creating-your-site.md

```diff
@@ -183,10 +184,25 @@ Point your browser to [localhost:8000][live preview] and you should see:
 When you're finished editing, you can build a static site from your Markdown
 files with:
 
-```
+``` sh
 mkdocs build
 ```
 
+If you're running Material for MkDocs from within Docker, use:
+
+=== "Unix"
+
+    ```
+    docker run --rm -v ${PWD}:/docs squidfunk/mkdocs-material build
+    sudo chown -R $UID:$UID site/
```

Comment:
Why is the `chown` necessary at this point? I'd say we drop it because otherwise some users might be confused.

Reply:
I've just tested it, and Docker it looks like the user/group running the Docker command is just forwarded to Docker, see:

```
> docker run --rm -it -p 8000:8000 -v ${PWD}:/docs squidfunk/mkdocs-material build
INFO     -  Cleaning site directory
INFO     -  Building documentation to directory: /docs/site
INFO     -  Documentation built in 0.81 seconds
> ls -lha site                         
total 80
drwxr-xr-x  11 squidfunk  staff   352B 10 Nov 16:11 .
drwxr-xr-x  13 squidfunk  staff   416B 10 Nov 16:11 ..
-rw-r--r--   1 squidfunk  staff   9,6K 10 Nov 16:11 404.html
drwxr-xr-x   5 squidfunk  staff   160B 10 Nov 16:11 assets
drwxr-xr-x   4 squidfunk  staff   128B 10 Nov 16:11 bar
-rw-r--r--   1 squidfunk  staff   279B 10 Nov 16:11 custom.css
drwxr-xr-x   3 squidfunk  staff    96B 10 Nov 16:11 foo
-rw-r--r--   1 squidfunk  staff    13K 10 Nov 16:11 index.html
drwxr-xr-x   3 squidfunk  staff    96B 10 Nov 16:11 search
-rw-r--r--   1 squidfunk  staff   609B 10 Nov 16:11 sitemap.xml
-rw-r--r--   1 squidfunk  staff   197B 10 Nov 16:11 sitemap.xml.gz
```

Reply:
Hmm, different here. I am using Linux Mint, which is an Ubuntu fork.

``` bash
$ docker run --rm -v ${PWD}:/docs squidfunk/mkdocs-material build # fresh+clean project
INFO     -  Cleaning site directory
INFO     -  Building documentation to directory: /docs/site
INFO     -  Documentation built in 0.19 seconds
$ ls -l site/
insgesamt 68
-rw-r--r-- 1 root root  8092 Nov 10 16:14 404.html
drwxr-xr-x 5 root root  4096 Nov 10 16:14 assets
-rw-r--r-- 1 root root 10813 Nov 10 16:14 index.html
drwxr-xr-x 2 root root  4096 Nov 10 16:14 search
-rw-r--r-- 1 root root   234 Nov 10 16:14 sitemap.xml
-rw-r--r-- 1 root root   188 Nov 10 16:14 sitemap.xml.gz
```